crates/matrix-sdk-ui/src/timeline/inner/mod.rs

```diff
@@ -587,8 +597,11 @@ impl<P: RoomDataProvider> TimelineInner<P> {
 
         let new_item = item.with_inner_kind(local_item.with_send_state(EventSendState::NotSentYet));
         let content = item.content.clone();
-        state.items.remove(idx);
-        state.items.push_back(new_item);
+
+        let mut txn = state.items.transaction();
+        txn.remove(idx);
+        txn.push_back(new_item);
+        txn.commit();
```

Comment:
Ditto, this one doesn't seem needed.

Reply:
This one is an `ObservableVectorTransaction`, which was always rollback-on-drop as far as the main branch / crates.io releases of eyeball-im go.

Reply:
I see (I find it a bit strange, but I would've found it weird that the other option was the default too, so 🤷).